### PR TITLE
index documents can be blacklisted

### PR DESCRIPTION
**Pull request by emig**
- is posible to blacklist index documents via configuration
- log info about configured blackliste
- requests related a blacklisted document return 422 (blacklisted document)

### CODE REVIEW

**Comment by emig** [comment]
@janl ping :)

**Comment by janl** [comment]
On it :)

File: src/main/java/com/github/rnewson/couchdb/lucene/couchdb/Couch.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public final JSONObject getInfo() throws IOException, JSONException {
     }
 
     public Database getDatabase(final String dbname) throws IOException {
-        return new Database(httpClient, url + dbname);
+        return new Database(httpClient, url, dbname);
```
Comment (janl) [inline src/main/java/com/github/rnewson/couchdb/lucene/couchdb/Couch.java:52]: nice

File: README.md [hunk 2]
```diff
@@ -649,6 +649,15 @@ and couchdb-lucene will authenticate to couchdb.
...
 A couple of 'expert' options can be set in the couchdb-lucene.ini file;
 
+CouchDB indexing documents can be blacklisted. This is useful when using 2 C-L instances,
+one indexing a new version of an index, allowing the old one to get the latest updates while the new one.
+is being indexed.
+<pre>
+[couchdb]
+blacklist=gin_call_centre/_design/search-version:0.0.5
+blacklist=gin_call_centre/_design/search-version:0.0.6
+</pre>
```
Comment (janl) [inline README.md:659]: Since this is loaded into a single CouchDB instance, how does either c-l instances which blacklist to use?

Comment (janl) [inline README.md:659]: cleared up in person, this file is not read by CouchDB, but by c-l itself.

**Comment by janl** [comment]
looks good to me!

File: README.md (hunk 2 above)
Comment (emig) [inline README.md:659]: Every instance loads the config file from its folder installation, so no risk of collision

**Pull request merged**